src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Rank {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```

언더스코어를 추가하여 가독성을 향상시킬 수 있습니다.